spec/requests/api/v0/uni_can_pay_for_subscription_spec.rb

```diff
@@ -8,6 +8,7 @@
   let(:university) { create(:user, role: :university) }
   let(:credentials) { university.create_new_auth_token }
   let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(credentials) }
+  
```

Comment:
Empty row

Reply:
Thanks!  I just talked to Dash and I think we may have to scrap this PR. . .Thomas had a discussion with her this weekend and said we were approaching this the wrong way.  To be continued. . .I'll fix this meanwhile.

Reply:
Well, the big question is why you add this column? How will that fit into the bigger picture?

Reply:
@tochman I'm trying to get the team on board with this now (we need to fix our user story and vote) but basically we'll use something similar to your demo to get coords to pop up on the main page. . .use OpenCage to turn those coords into City/Country . . .then sort the articles on the main page under something like "Local Research" where the front end coords will be compared against the country/city column on the back end and be displayed.